Foundation/src/Environment_WIN32U.cpp

```diff
@@ -16,16 +16,14 @@
 #include "Poco/Exception.h"
 #include "Poco/UnicodeConverter.h"
 #include "Poco/Buffer.h"
+
 #include <sstream>
 #include <cstring>
 #include <memory>
-#include "Poco/UnWindows.h"
```

Comment:
@andrewauclair why remove this? the main purpose of that header is to reduce windows bloat through definition of `WIN32_LEAN_AND_MEAN`

Reply:
I ended up starting the PR #4405 code from @matejk. I can put that back if you want.